relayer/relayer.go

```diff
@@ -161,55 +165,64 @@ func NewRelayer(
 }
 
 // Helper to process missed blocks. Chooses as the starting block the maximum of the latest processed block and the configured start block height.
-// startBlockHeight cannot be nil
-func (r *Relayer) processMissedBlocks(
+// If the DB does not contain a value, startBlockHeight cannot be nil
+func (r *Relayer) setProcessedBlockHeight(
```

Comment:
Wasn't introduced in this PR, but the `error` should be the final return value to keep consistent with conventions.